core/src/services/routing.js

```diff
@@ -123,20 +123,30 @@ const escapeRegExp = string => {
   return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
 };
 
-const replaceVars = (viewUrl, params, prefix) => {
+const replaceVars = (viewUrl, params, prefix, noParenthesis) => {
```

Comment:
If my tests were correct here we are replacing just the second dynamic parameter. If this is true, where are we replacing the first dynamic parameter and why aren´t we replacing all parameters at once in the same place?

Reply:
Could you please explain this issue in detail so that we make sure this is fixed in that ticket?

Reply:
It is explained in the related issue #213.